autogen/coding/local_commandline_code_executor.py

```diff
@@ -193,12 +193,12 @@ def restart(self) -> None:
 
 # From stack overflow: https://stackoverflow.com/a/52087847/2214524
 class _DeprecatedClassMeta(type):
-    def __new__(cls, name, bases, classdict, *args, **kwargs):
+    def __new__(cls, name, bases, classdict, *args, **kwargs):  # type: ignore
```

Comment:
I believe this should work:
```suggestion
    def __new__(cls, name, bases, classdict, *args, **kwargs) -> "_DeprecatedClassMeta":
```

Reply:
I appreciate these suggestions but all the args would need types too. I opted to ignore these as it's a temporary class only used to provide the warning for a little bit. We'll remove it when we remove the old name. It wasn't even documented or anything so I don't think we will keep it around for long